engine/src/flutter/docs/Custom-Flutter-Engine-Embedders.md

```diff
@@ -12,7 +12,7 @@ This is a very low level API and is not suitable for beginners.
     * The binary is not stripped and contains debug information. Embedders are advised to strip the binary before deployment.
   * The Windows buildbot uploads the artifacts to a known location. Access it here [`https://storage.googleapis.com/flutter_infra_release/flutter/FLUTTER_HASH/windows-x64/windows-x64-embedder.zip`](https://storage.googleapis.com/flutter_infra_release/flutter/e672b006cb34c921db85b8e2f482ed3144a4574b/windows-x64/windows-x64-embedder.zip)
     * Replace `FLUTTER_HASH` with the SHA of the Flutter commit you wish to use.
-  * You can also obtain that SHA from the [`engine.version`](https://github.com/flutter/flutter/blob/main/bin/internal/engine.version) file in your Flutter framework checkout. This allows you to exactly match the engine version with the Flutter framework version.
```

Comment:
Can you reword this to take the SHA of the flutter/flutter repo instead?

Reply:
Is this whole line then redundant - the previous use of "the SHA of the Flutter commit you wish to use" is sufficient?

Reply:
Yeah, you're right.

Reply:
Removed!